torch/onnx/_internal/fx/type_utils.py

```diff
@@ -38,6 +43,23 @@ def from_torch_dtype_to_onnx_dtype_str(dtype: Union[torch.dtype, type]) -> Set[s
     return _TORCH_DTYPE_TO_COMPATIBLE_ONNX_TYPE_STRINGS[dtype]
 
 
+def from_python_type_to_onnx_attribute_type(
+    dtype: type,
+) -> Optional[onnx.defs.OpSchema.AttrType]:
+    import onnx.defs  # type: ignore[import]
+
+    _PYTHON_TYPE_TO_ONNX_ATTRIBUTE_TYPE = {
```

Comment:
bool?

Reply:
Bool is actually not an AttributeProto type

https://github.com/onnx/onnx/blob/547032edb8f86adb1b38d37f0a87aa61ee2ff580/onnx/onnx.proto#L118-L139

Reply:
bool -> onnx.defs.OpSchema.AttrType.INT

Reply:
I am surprised that we don't have any test cases using bool attribute.

Reply:
Dropout maybe?

Reply:
I can try adding it as a test case here

Reply:
Added a test in `test_fx_to_onnx.py` showing that although attribute type checking is added, the aten::convolution which has sequence/bool type in attribute can still be found as perfect match.